USING_DOCKER.md

```diff
@@ -0,0 +1,33 @@
+# Covalent Docker Guide
```

Comment:
This guide should be linked to from the README, or included directly in the README since it's nice and short. Same for readthedocs.

Reply:
@AlejandroEsquivel was this linked? I didn't see a link in the README but maybe I missed it

Reply:
No I don't think it's linked, I actually intended to delete this

Reply:
ah OK I think deleting it is fine. should we include some Docker information in the README or do you think having it in getting started is enough?